frontend/src/pages/index.js

```diff
@@ -0,0 +1,9 @@
+export { Admin } from "./Admin";
```

Comment:
There should always be some type of comment in every file, to explain what its purpose is.  Although i kind of get it by the context, still might be a good idea to through something in there.